Scripts/build-phases/swiftlint.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#
+# Runs SwiftLint on the whole workspace
+#
+# This does not run in Continuous Integration.
+#
+
+# Abort if we are running in CI
+# See https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables
+if [ "$CI" = true ] ; then
```

Comment:
+1 for using `$CI`, which seems to be the standard env var most CI providers use. That is, if we'll move away from Circle CI, we won't have to update this script 🙌.

Reply:
How do you feel about echoing something like the following, just to add a note about it in the CI logs?

```
echo "warning: skipping SwiftLint build phase because running on CI."
```

_I swear I saw someone use something different from warning or error to show just an info icon, but I can't find the syntax_ 😞

Reply:
Thanks! Added in 1896de78f06f9bf290bbe4e3d3ef6b37971d3d22.